src/s3ql/backends/gs.py

```diff
@@ -529,7 +534,7 @@ def _get_access_token(self):
 
         if self.adc:
             try:
-                self.adc.refresh()
+                self.adc.refresh(GAuthHTTPRequestor(self.ssl_context))
```

Comment:
I would have hoped that the adc object saves the requestor instance - is this really not the case?

Reply:
Passing None did not work In this case, I can check the code tomorrow.